app/assets/javascripts/angular.app.js.erb

```diff
@@ -57,14 +57,14 @@ app.run(['Angularytics', '$http', function(Angularytics, $http) {
   Angularytics.init();
 
   // First load, set analytics custom dimensions: user_id and school
-  $http.get('/settings.json').success(function(settings) {      
-    
-    // Analytics maintinas these internal names for these custom dimensions:
+  $http.get('/settings.json').success(function(settings) {
+
+    // Analytics maintains these internal names for these custom dimensions:
```

Comment:
Didn't you get rid of this information in another PR?

Reply:
@nnat425 Yes, but that one may not be merged in soon, because it's not a priority to finish.